src/js/App.js

```diff
@@ -0,0 +1,2 @@
+import '../style.css';
+import './todo.js';
```

Comment:
- [x] Great job 👍 with your project and I suggest for you merge the App js with Todo js because I have noticed that There is nothing a lot in the App, so I suggest 😊 merging the app todo, but you are almost there.

Reply:
Thanks alot for your comment